lib/resty/balancer/utils.lua

```diff
@@ -0,0 +1,53 @@
+--
+-- first try use luajit's function
```

Comment:
The comment is confusing...
Is it suitable to appear at the top of this file?

Reply:
i will remove this comment.